Intersect.Editor/Core/Graphics.cs

```diff
@@ -1348,7 +1348,9 @@ bool alternate
         var x2 = Options.Instance.Map.MapWidth;
```

Comment:
`Options.Instance.Map.MapWidth` should be cached and reused in the instances after this